src/collection.js

```diff
@@ -29,6 +29,17 @@ export let insertAtIndex = _.curry((index, val, collection) =>
     : insertAtArrayIndex(index, val, collection)
 )
 
-export let compactMap = _.curry((fn, collection) =>
-  _.flow(_.map(fn), _.compact)(collection)
-)
+export let compactMap = _.curry((iteratee, collection) => {
```

Comment:
Can we do a quick benchmark? Lodash supports shortcut fusion which should be effectively the same thing. My first feedback was gonna be to do this with transducers https://runkit.com/daedalus28/transducer-example but lodash does this automatically in some cases - might only be only explicit `chain` calls though.

Reply:
Fusion is only supported in the chain scenario. https://lodash.com/docs/4.17.15#lodash